charts/templates/clusterrole.yaml

```diff
@@ -3,21 +3,11 @@ kind: ClusterRole
 metadata:
   name: {{ include "cloudtty.controllerManager.fullname" . }}
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - services
-  verbs: ["*"]
-- apiGroups:
-  - ""
-  resources:
-  - nodes
-  verbs: ["get", "watch", "list"]
-- apiGroups:
-  - batch
-  resources:
-  - jobs
-  verbs: ["*"]
+- apiGroups: ['*']
```

Comment:
I‘ve concern here.
The purpose of `clusterrole` is for the usage of `operator` code, instead of tty.
Those RBAC was generated by `kubebuilder` comments in code (example: `//+kubebuilder:rbac:groups=.....`)

aka, I feel a bit unconformable with this change.

Reply:
You are right, it does seem to make sense. let me implement that again.

My approach may be simple and crude. 😄